jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -272,6 +285,8 @@ YUI.add('store', function() {
               <h2>Kubernetes</h2>
               {kubernetesButton}
             </div>
+            <span onClick={this._handleSearchClick}
```

Comment:
There was a strange issue that events from the images weren't bubbling to the box. I can investigate further, but this Just Works &trade;